drivers/storage/digitalocean/storage/digitalocean_storage.go

```diff
@@ -0,0 +1,336 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_digitalocean
+
+package storage
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/digitalocean/godo"
+
+	gofig "github.com/akutz/gofig/types"
+	"github.com/akutz/goof"
+	"github.com/codedellemc/libstorage/api/context"
+	"github.com/codedellemc/libstorage/api/registry"
+	"github.com/codedellemc/libstorage/api/types"
+	do "github.com/codedellemc/libstorage/drivers/storage/digitalocean"
+	doUtils "github.com/codedellemc/libstorage/drivers/storage/digitalocean/utils"
+)
+
+type driver struct {
+	name   string
+	config gofig.Config
+	client *godo.Client
+}
+
+func init() {
+	registry.RegisterStorageDriver(do.Name, newDriver)
+}
+
+func newDriver() types.StorageDriver {
+	return &driver{name: do.Name}
+}
+
+func (d *driver) Name() string {
+	return do.Name
+}
+
+func (d *driver) Init(ctx types.Context, config gofig.Config) error {
+	d.config = config
+	token := d.config.GetString(do.ConfigDOToken)
+
+	fields := log.Fields{
+		"token": token,
+	}
+
+	if token == "" {
+		fields["token"] = ""
+	} else {
+		fields["token"] = "******"
+	}
+
+	fields["region"] = d.config.GetString("digitalocean.region")
```

Comment:
Hi @protochron,

You should use the constants declared in `github.com/codedellemc/libstorage/drivers/storage/digitalocean` instead of free-form strings in order to prevent possible error. You could even consider writing helper functions such as [those](https://github.com/codedellemc/libstorage/blob/master/drivers/storage/ebs/storage/ebs_storage.go#L1321-L1394) found in the EBS driver